dice/apps/games/tests/test_views/test_room_views.py

```diff
@@ -32,21 +34,22 @@ def test_create_room(self):
         rooms_before = Room.objects.count()
         response = self.client.post('/api/v1/rooms/')
         rooms_after = Room.objects.count()
-        self.assertEqual(response.status_code, 201)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         self.assertEqual(rooms_after, rooms_before + 1)
         host_id = response.json().get('host')
         self.assertEqual(host_id, self.user.id)
 
     def test_create_room_user_in_two_rooms(self):
-        """Ensure http 403 is returned when user is in active room."""
+        """Ensure http 409 is returned when user is in active room."""
 
         room = RoomFactory()
         room.host = self.user
         room.save()
         rooms_before = Room.objects.count()
         response = self.client.post('/api/v1/rooms/')
         rooms_after = Room.objects.count()
-        self.assertEqual(response.status_code, 403)
+        self.assertEqual(response.status_code, status.HTTP_409_CONFLICT)
+        self.assertEqual(response.json().get('message'), "You are a member of other room")
```

Comment:
NIP: it's a test, so you could access `dict` item explicitly:

```suggestion
        self.assertEqual(response.json()['message'], "You are a member of other room")
```

If `response.json()` will not contain `key` then it's an error, so you will get a `KeyError` exception (which produces a more readable and obvious error) and the test will fail as expected.